dev/tools/gen_defaults/data/menu.json

```diff
@@ -8,8 +8,6 @@
   "md.comp.menu.container.shadow-color": "shadow",
   "md.comp.menu.container.shape": "md.sys.shape.corner.extra-small",
   "md.comp.menu.container.surface-tint-layer.color": "surfaceTint",
-  "md.comp.menu.divider.color": "surfaceVariant",
-  "md.comp.menu.divider.height": 1.0,
```

Comment:
cc: @guidezpl @QuncCccccc 

Hey!  Any reason divider tokens are removed? 
I added them in my PR https://github.com/flutter/flutter/pull/103606 but now when updating the PR I need to noticed this.